examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywall/PaywallScreenViewModel.kt

```diff
@@ -35,6 +39,42 @@ class PaywallScreenViewModelImpl(
         updateOffering()
     }
 
+    override fun onPurchaseCompleted(customerInfo: CustomerInfo, storeTransaction: StoreTransaction) {
+        val value = _state.value
+        if (value is PaywallScreenState.Loaded) {
+            _state.update {
+                value.copy(
+                    displayCompletedPurchaseMessage = true,
+                    displayErrorPurchasingMessage = false,
+                )
+            }
+        }
+    }
+
+    override fun onPurchaseError(error: PurchasesError) {
```

Comment:
I guess we are not displaying the error message in the dialog right? I wonder if instead of a boolean, we should use a nullable string in the state so we can display the error message.

Reply:
I see we already made the conversion to a nullable string in the followup PR... So then the question is whether we want to include the error in the dialog... IMO, it might be a good idea for a tester app since it would be easier to visualize errors in case they happen when downloaded from the play store.

Reply:
Added error message